tor/controller.go

```diff
@@ -77,6 +78,14 @@ var (
 	// errTCNotStarted is used when we require the tor controller to be
 	// not stopped while it is.
 	errTCStopped = errors.New("tor controller must not be stopped")
+
+	// replyFieldRegexp is the regular expression used to find fields in a reply.
```

Comment:
nit: comments are slightly longer than 80 chars when using 8 spaces for the tab character.

Reply:
Oh, I missed the 8 space tab width, fixed.